src/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render } from "@testing-library/react";
+import GenericSeqPanel from "./GenericSeqPanel";
+
+test("expect protein rendering", async () => {
+  const { findByText } = render(
+    <GenericSeqPanel
+      refseq="X"
+      start={13201770}
+      end={13216729}
+      gene="WBGene00006749"
+      transcript="R12H7.1a.2"
+      mode="protein"
+      nclistbaseurl="https://s3.amazonaws.com/agrjbrowse/MOD-jbrowses/WormBase/WS287/c_elegans_PRJNA13758/"
+      urltemplate="tracks/Curated_Genes/{refseq}/trackData.jsonz"
+      fastaurl="https://s3.amazonaws.com/wormbase-modencode/fasta/current/c_elegans.PRJNA13758.WS284.genomic.fa.gz"
+    />
+  );
+  await findByText(/R12H7.1a.2-protein/, {}, { timeout: 15000 });
```

Comment:
this test (index.test.tsx) basically renders the component just like the "app" is doing (the index.tsx). technically this is fetching data from an external URL which is not the best situation for tests, better to use local data, but that can be tricky, so it might be ok to do this

Reply:
Yeah, I thought about the fact that testing with remote data is fraught. But I also haven't really written the component to allow for local data access and didn't really want to take the time to add it in (I added it to the section on future work thought).  Also, I didn't realize `findByText` was an available method, though I think I still like the approach of specifiying all of location, gene and transcript name to get the features that are needed.